perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_unixGID_namespace.java

```diff
@@ -283,12 +284,12 @@ public void changedAttributeHook(PerunSessionImpl session, Group group, Attribut
 		return dependencies;
 	}
 
-	/*public AttributeDefinition getAttributeDefinition() {
+	public AttributeDefinition getAttributeDefinition() {
 		AttributeDefinition attr = new AttributeDefinition();
 		attr.setNamespace(AttributesManager.NS_GROUP_ATTR_DEF);
 		attr.setFriendlyName("unixGID-namespace");
 		attr.setType(Integer.class.getName());
 		attr.setDescription("Unix GID namespace.");
 		return attr;
-	}*/
+	}
```

Comment:
Why did you remove comments here? It is there because of namespace in the module.